python/lsst/meas/astrom/matchOptimisticB.py

```diff
@@ -3,25 +3,20 @@
 from builtins import object
 import math
 
-import numpy as np
-
 import lsst.afw.table as afwTable
```

Comment:
Sublimetext is what caught it for me. I don't like committing a file with any warning dots at all. You're using SublimeLinter+flake8?